submissions/expero1/js-dom/script.js

```diff
@@ -0,0 +1,82 @@
+const createElem = (elem, attrObj, className) => {
+  elem = document.createElement(elem);
+  elem.classList.add(className);
+  for (attrName in attrObj) {
+    elem.setAttribute(attrName, attrObj[attrName]);
+  }
+  return elem;
+};
+
+const createMenu = (data) => {
+  let menu = createElem("ul", { id: "menu", class: "menu" });
+  data.forEach(
+    (item, index) => { menu.appendChild(createMenuItem(data[index], index)) }
+  )
+  return menu;
+};
+
+const createMenuItem = (data, id) => {
```

Comment:
`data` is too general naming. 
In this case you should give name your parameter `menuItemData`, `item` etc. (This is just example you can create more clear and cool name)